README.md

```diff
@@ -202,9 +217,23 @@ variables.
     have the form
     `http(s)://<externally_reachable_app_url>/nhs-login-callback`.
 
-  - `AWS_DYNAMODB_SUBMISSIONS_TABLE_NAME` **\[required\]**: This
-    variable contains the table name that the app will submit form
-    responses to.
+  - `AWS_RDS_DATABASE_NAME` **\[required\]**: This variable sets the
+    database that the app will use to persist form answers.
+
+  - `AWS_RDS_DATABASE_SCHEMA` **\[required\]**: this variable sets the
```

Comment:
can this be blank for prod?

Reply:
It can be set to a blank string, but it is currently set as a required variable, which is not super-logical! Changed.

Reply:
Might be best to remove this as its not supported 
https://docs.aws.amazon.com/rdsdataservice/latest/APIReference/API_ExecuteStatement.html

Reply:
Good spot 👍